lib/helpers/logic.dart

```diff
@@ -1,10 +1,12 @@
 import 'dart:collection';
 import 'dart:math';
 
+import 'package:spelling_bee/blocs/game_bloc.dart';
```

Comment:
This creates circular imports. Either 
(a) the constant on line 9 can be a string or 
(b) the Message class can be moved from the other file to here. 
But I like option (a) for being consistent with the other use case of _setTempMessage